rpc/grpc/grpc_test.go

```diff
@@ -7,27 +7,36 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/tendermint/tendermint/abci/example/kvstore"
-	"github.com/tendermint/tendermint/node"
+	"github.com/tendermint/tendermint/config"
+	"github.com/tendermint/tendermint/libs/service"
 	core_grpc "github.com/tendermint/tendermint/rpc/grpc"
 	rpctest "github.com/tendermint/tendermint/rpc/test"
 )
 
-func NodeSuite(t *testing.T) *node.Node {
+func NodeSuite(t *testing.T) (service.Service, *config.Config) {
```

Comment:
Seeing as this function gets repeated a couple of times, why not add it to `rpc/test/helpers.go`

Reply:
the rpc/client version uses a persistentkv application and the others don't. I'm inclined to leave it the way it is now and clean it up later as part of more/deeper thinking we should do about the RPC layer